src/cpg_seqr_loader/scripts/alphagenome_only_scores.py

```diff
@@ -0,0 +1,249 @@
+    P.add_argument('--output-dir', default='gs://cpg-kidgen-test/alphagenome_scan_plots', help='Dir for plot images.')
+
+    return P
+
+
+# ------------------------------------------------------------------
+# I/O
+# --------------------------------------------
+
+
+# ------------------------------------------------------------------
+# Model / Annotation
+# ------------------------------------------------------------------
+
+
+
+api_key = config.config_retrieve(['alphagenome_params', 'api_key'], default=None)
+organ = config.config_retrieve(
+    ['alphagenome_params', 'organs'],
+    default=[
+        'UBERON:0000992',
+        'UBERON:0002371',
+        'UBERON:0000948',
+        'UBERON:0000955',
+        'UBERON:0001134',
+        'UBERON:0001264',
+    ],
+)
+
+
+def get_dna_model(api_key: str | None = None):
+    if api_key is None:
+        try:
+            api_key = colab_utils.get_api_key()
+        except Exception as e:
+            # If not Colab
+            print(f'Error getting API key from Colab: {e}')
+            api_key = ''  # Insert your API key here if not working
+    return dna_client.create(api_key)
+
+
+
+
+
+def write_table(df: pd.DataFrame, path: str):
+    ext = Path(path).suffix.lower()
+    if ext in {'.tsv', '.txt'}:
+        df.to_csv(path, sep='\t', index=False)
+    elif ext in {'.xlsx', '.xls'}:
+        df.to_excel(path, index=False)
+    else:
+        # default csv
+        df.to_csv(path, index=False)
+
+def check_gcs_path(gcs_path: str) -> str:
+    """
+    Check if the provided GCS path starts with 'b/' and convert it to 'gs://' format.
+    """
+    if gcs_path.startswith("b/"):
+        print('Detected GCS path starting with "b/". Converting to gs:// format.')
+        # convert from b/.../o/... to gs://bucket/object
+        parts = gcs_path.split("/")
+        bucket = parts[1]
+        obj = "/".join(parts[3:])
+        return f"gs://{bucket}/{obj}"
+    return gcs_path
+
+
+def main(argv: list[str] | None = None) -> int:
+    args = build_parser().parse_args(argv)
+    gcs_path = args.variants
+    gcs_path = check_gcs_path(gcs_path)
+    fs = gcsfs.GCSFileSystem()
+    with fs.open(gcs_path, "rb") as f:
+        variants_df = pd.read_csv(f, sep="\t")
+
+    #transcript_extractor = #load_transcript_extractor(gtf)
+    dna_model = get_dna_model(api_key)
+    organism = 'human'  # @param ["human", "mouse"] {type:"string"}
+    organs = organ or [
+        'UBERON:0000992',
+        'UBERON:0002371',
+        'UBERON:0000948',
+        'UBERON:0000955',
+        'UBERON:0001134',
+        'UBERON:0001264',
+    ]
+    #out_dir = to_anypath(args.output_dir)
+    sequence_length = '1MB'  # @param ["2KB", "16KB", "100KB", "500KB", "1MB"] { type:"string" }
+    sequence_length = dna_client.SUPPORTED_SEQUENCE_LENGTHS[
+        f'SEQUENCE_LENGTH_{sequence_length}'
+    ]
+
+    # @markdown Specify which scorers to use to score your variants:
+    score_rna_seq = True  # @param { type: "boolean"}
```

Comment:
This isn't correct syntax for type hinting, in python this would be

> score_rna_seq: bool = True

https://docs.python.org/3/library/typing.html

For Boolean variables the type hint is implicit, the variable has a Strict type. Hinting would be more important if you were making an empty list or a set, and static code inspection wouldn't know what type of object you intended to put in it